relay-config/src/config.rs

```
@@ -2397,9 +2395,8 @@ impl Config {
     /// Returns aggregator config for a given metrics namespace.
     pub fn aggregator_config_for(&self, namespace: MetricNamespace) -> &AggregatorServiceConfig {
         for entry in &self.values.secondary_aggregators {
-            match entry.condition {
-                Condition::Eq(Field::Namespace(ns)) if ns == namespace => return &entry.config,
-                _ => (),
```

This was actually incredibly dangerous -- there is no warning that we silently fall back to a wrong aggregator config. We should not add such blanked match arms, especially not if the enum is not exhaustive.

---

👍 agreed.